pylintrc

```diff
@@ -332,7 +332,7 @@ max-locals=25
 max-returns=11
 
 # Maximum number of branch for function / method body
-max-branches=26
+max-branches=27
```

Comment:
This number is quite arbitrary but I'm not sure if we should change it. I'll leave this decision to others though.

Reply:
I checked the blame; it was just number of the "branchiest" method at the time.